jxl/src/image.rs

```diff
@@ -442,18 +442,18 @@ impl<'a, T: ImageDataType> ImageRectMut<'a, T> {
 
     #[instrument(skip_all)]
     pub fn copy_from(&mut self, other: ImageRect<'_, T>) -> Result<()> {
```

Comment:
Could also make the rect smaller instead of changing this :-)

Reply:
Done.